tests/NHSD.BuyingCatalogue.Ordering.Services.UnitTests/OrderServiceTests.cs

```diff
@@ -0,0 +1,232 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using NHSD.BuyingCatalogue.Ordering.Api.UnitTests.AutoFixture;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+using NHSD.BuyingCatalogue.Ordering.Persistence.Data;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Ordering.Services.UnitTests
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    [SuppressMessage("ReSharper", "NUnit.MethodWithParametersAndTestAttribute", Justification = "False positive")]
+    internal static class OrderServiceTests
+    {
+        [Test]
+        public static void Constructor_NullAccessor_ThrowsArgumentNullException()
+        {
+            Assert.Throws<ArgumentNullException>(() => _ = new OrderingPartyService(null));
+        }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static async Task GetOrder_ReturnsNull(
+            CallOffId callOffId,
+            OrderService service)
+        {
+            var result = await service.GetOrder(callOffId);
+
+            result.Should().BeNull();
+        }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static async Task GetOrder_ReturnsOrder(
+            [Frozen] ApplicationDbContext context,
+            Order order,
+            OrderService service)
+        {
+            context.Order.Add(order);
+            await context.SaveChangesAsync();
+
+            var expectedResult = order;
```

Comment:
I don't think we really need this line. I think we either just use `order` in `result.Should().Be(order)` or we can rename `order` to `expectedResult`. The first option might be better as it's more consistent.

Reply:
Updated `expectedResult `and updated `assert`